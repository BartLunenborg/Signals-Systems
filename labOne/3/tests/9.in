1234 1 3 2 -0.1415926536
