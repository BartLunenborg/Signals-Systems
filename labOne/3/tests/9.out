x(t)=1.00*cos(2*pi*1234.00*t-0.14)
